repo_health/__init__.py

```diff
@@ -0,0 +1,51 @@
+"""
+Enforce selected packaging guidelines.
```

Comment:
Maybe update the docstring here?  It doesn't really enforce anything as written, and as the top-level package this should probably give a little more context about the package as a whole.